### PR TITLE
feat(ui5-select): Add value state message

### PR DESCRIPTION
**Pull request by nnaydenow**
Now, **ValueStateMessage**s are supported in **ui5-select** when **ValueState** is set to "Warning", Error or "Information"

#1086

### CODE REVIEW

File: packages/main/src/SelectPopover.hbs [hunk 1]
```diff
@@ -21,8 +21,23 @@
 				>
 				</ui5-button>
 			</div>
+			{{#if _isPhone}}
+				{{#if hasValueStateText}}
+					<div class="{{classes.popoverValueState}} row" style="{{styles.dialogHeader}}">
```
Comment (vladitasev) [inline packages/main/src/SelectPopover.hbs:26]: Is this "row" class real or a leftover?

File: packages/main/src/Select.js [hunk 2]
```diff
@@ -504,11 +540,87 @@ class Select extends UI5Element {
...
+		return {
+			popoverHeader: {
+				"width": `${this.offsetWidth}px`,
+			},
+			responsivePopoverHeader: {
+				"display": this.options.length && this._listWidth === 0 ? "none" : "inline-block",
+				"width": `${this.options.length ? this._listWidth : this.offsetWidth}px`,
+				"padding": "0.5625rem 1rem",
+			},
+			dialogHeader: {
```
Comment (vladitasev) [inline packages/main/src/Select.js:565]: Can't this be "class", usually in "styles" we only set dynamic properties

File: packages/main/src/Select.js [hunk 3]
```diff
@@ -504,11 +540,87 @@ class Select extends UI5Element {
...
+
+	get styles() {
+		return {
+			popoverHeader: {
+				"width": `${this.offsetWidth}px`,
+			},
+			responsivePopoverHeader: {
+				"display": this.options.length && this._listWidth === 0 ? "none" : "inline-block",
+				"width": `${this.options.length ? this._listWidth : this.offsetWidth}px`,
+				"padding": "0.5625rem 1rem",
```
Comment (vladitasev) [inline packages/main/src/Select.js:563]: Same here, move this to a class if possible

File: packages/main/src/SelectPopover.hbs (hunk 1 above)
Comment (nnaydenow) [reply packages/main/src/SelectPopover.hbs]: This is real class which comes from ui5-resposnive-popover. It is used here to set correct font-size and other things to align the value state message to dialog's styles on mobile.

File: packages/main/src/Select.js (hunk 2 above)
Comment (nnaydenow) [reply packages/main/src/Select.js]: Done

File: packages/main/src/Select.js (hunk 3 above)
Comment (nnaydenow) [reply packages/main/src/Select.js]: Done

**Pull request merged**